src/pyjion/dis.py

```diff
@@ -664,11 +664,32 @@ def dis(f):
 
 
 def dis_native(f):
+
     try:
         import distorm3
     except ImportError:
         raise ModuleNotFoundError("Install distorm3 before disassembling native functions")
+
     code, code_length, position = dump_native(f)
     iterable = distorm3.DecodeGenerator(position, bytes(code), distorm3.Decode64Bits)
-    for (offset, size, instruction, hexdump) in iterable:
-        print("%.8x: %s" % (offset, instruction))
+
+    disassembled = [(offset, instruction) for (offset, size, instruction, hexdump) in iterable]
+
+    try:
+        from rich.console import Console
```

Comment:
Move the rich imports into the same try block as the `distorm3` try block and update the message instead please